contents/dataframes.md

```diff
@@ -1,3 +1,228 @@
 # DataFrames.jl {#sec:dataframes}
 
-Placeholder
+Most data is formatted as a table.
+By a table, we mean that the data consists of rows and columns, and the columns are usually of the same data type, whereas rows have different types.
+The rows, in practise, denote measurements and columns denote variables.
+For example, we can have a table of tv shows, which country it was produced, and our own rating for the shows, see @tbl:tv_shows.
+
+```{=comment}
+Using a different example from the rest in the chapter to make the text a bit more interesting.
+We could even ask the reader to answer the queries described below as exercises.
+```
+
+```jl
+tv_shows = DataFrame(
+    name=["Game of Thrones", "The Crown", "Friends", "..."],
+    country=["America", "England", "America", "..."],
+    rating=[8.2, 7.3, 7.8, "..."])
+Options(tv_shows; label="tv_shows")
+```
+
+Here, the dots mean that this could be a very long table and we only show a few rows.
+While doing research, questions on this data start to arise.
+For large tables, computers would be able to answer these kinds of questions much quicker than you could do it by hand.
+Examples of, so called _queries_, for this data could be:
+
+- Which show did I give the highest rating?
+- Which shows are made in America?
+- Which shows where in the same country?
```

Comment:
```suggestion
- Which TV shows were produced in the same country?
```
The question was ambiguous. Was this you intended to write? Sorry if I misunderstood.